streaming/base/storage/upload.py

```diff
@@ -171,8 +171,9 @@ def __init__(self,
                 raise FileExistsError(f'Directory is not empty: {self.local}')
```

Comment:
Can you please extend the error message saying that you set `exist_ok=True` if you would like to delete the content from the local directory and start fresh?

Reply:
Thanks for the quick response! Done.